frameworks/engineering/mobile.md

```diff
@@ -163,13 +163,13 @@ Check out the original mobile engineering progression site [here](https://eng-pr
     > Modularisation
 - Debugs complex problems across a variety of subject areas
   - *Seeks out technical knowledge to compliment their own, researches and patches third party libraries where needed. Uses reasoning to debug and find the root cause of issues. Articulates problem and proposed solution to both technical and non technical peers.*
-- RFCs and architects complex app flows
+- Writes proposals and architects complex app flows
```

Comment:
I replaced "RFCs" (as a verb) with "Writes proposals" as "proposes" by itself doesn't seem to imply this means writing down what we know as "proposal". Just saying "proposes" could be interpreted as just mentioning it somewhere verbally.